pkg/golinters/mnd/mnd.go

```diff
@@ -25,7 +25,9 @@ func NewGoMND(settings *config.GoMndSettings) *goanalysis.Linter {
 
 	if settings != nil && len(settings.Settings) > 0 {
 		// Convert deprecated setting.
-		linterCfg = settings.Settings
+		linterCfg = map[string]map[string]any{
+			a.Name: settings.Settings["mnd"],
```

Comment:
1. can you write the integration test (despite of linter deprecation status)
2. `gomnd` key for settings is not possible?

Reply:
1. because we will remove this code and the deprecation cannot be tested.
2. it's not possible.